src/EditorFeatures/Core/InlineHints/InlineHintsTag.cs

```diff
@@ -153,7 +132,9 @@ private static FrameworkElement CreateElement(
         ClassificationTypeMap typeMap,
         bool classify)
     {
+        // If we have an inline hint then we should have at least one line of text in the view.
```

Comment:
Fwiw I don't know if this is true. For example, if the user has inline hints and deletes all the lines of code in the file. It's possible some hints remain until we do another computation pass.